compiler/src/iree/compiler/Codegen/Common/GPU/GPUDistributionPatterns.cpp

```diff
@@ -597,6 +598,72 @@ struct DistributeScfFor final : OpDistributionPattern<scf::ForOp> {
   }
 };
 
+struct DistributeBroadcastLayoutAttr final
+    : OpDistributionPattern<vector::BroadcastOp> {
+  using OpDistributionPattern::OpDistributionPattern;
+
+  LogicalResult matchAndRewrite(vector::BroadcastOp broadcastOp,
+                                DistributionSignature &signature,
+                                PatternRewriter &rewriter) const override {
+
+    VectorValue source = dyn_cast<VectorValue>(broadcastOp.getSource());
+    if (!source) {
+      // TODO: Add support for scalar broadcasting.
+      llvm::errs() << "scalar??\n";
```

Comment:
Maybe more descriptive error message?

Reply:
Also do these error messages need to be inside some LLVM_DEBUG macro?

Reply:
oops. accidently left debug messages.